README.md

```diff
@@ -194,6 +194,26 @@ https://docs.ansible.com/ansible/latest/user_guide/vault.html for details.
 If these variables are set, `ha_cluster_regenerate_keys` is ignored for this
 certificate - key pair.
 
+#### `ha_cluster_pcsd_certificates`
+
+You can either use existing TLS certificate and private key files by providing
+them with [`ha_cluster_pcsd_public_key_src` and `ha_cluster_pcsd_private_key_src`](#ha_cluster_pcsd_public_key_src-ha_cluster_pcsd_private_key_src),
+or use the role to create certificates by providing `ha_cluster_pcsd_certificates`.
+
+Use this variable to generate certificate and private key for TLS encryption
+using the `fedora.linux_system_roles.certificate`.
+
+The value of `ha_cluster_pcsd_certificates` is set to the variable `certificate_requests`
+in the `certificate` role.
+For more information, see the `certificate_requests` section in the `certificate`
+role documentation.
+
+When you set this variable, you must not set `ha_cluster_pcsd_public_key_src` and
+`ha_cluster_pcsd_private_key_src` variables.
+
+If this variable is set, `ha_cluster_regenerate_keys` is ignored for this
+certificate - key pair.
+
```

Comment:
Do we want an example in `README.md` and an example playbook to show how the `ha_cluster_pcsd_certificates` variable is used?